app/src/main/java/me/hufman/androidautoidrive/carapp/music/views/BrowsePageView.kt

```diff
@@ -166,13 +159,33 @@ class BrowsePageView(val state: RHMIState, val musicImageIDs: MusicImageIDs, val
 				show()  // show the next page deeper
 				return@launch
 			} else {
-				currentListModel = object: RHMIListAdapter<MusicMetadata>(3, musicList) {
+				currentListModel = object: RHMIListAdapter<MusicMetadata>(4, musicList) {
 					override fun convertRow(index: Int, item: MusicMetadata): Array<Any> {
+						val checkmarkIcon = if (previouslySelected == item) checkmarkIcon else ""
+						val coverArt = item.coverArt
+						val coverArtImage =
+								if (coverArt != null)
```

Comment:
Can these if/else clauses have brackets, to match the rest of the code?

Reply:
Yeah sorry I must have missed that, updated to have brackets